las_geoh5_assets/uijson/import_las_files.ui.json

```diff
@@ -4,27 +4,14 @@
     "run_command": "las_geoh5.import_files.driver",
     "run_command_boolean": {
         "value": false,
-        "label": "Run python module",
-        "main": true,
-        "tooltip": "Warning: launches process to run python model on save"
+        "label": "Run python module ",
+        "tooltip": "Warning: launches process to run python model on save",
+        "main": true
     },
     "monitoring_directory": "",
-    "conda_environment": "las-geoh5",
+    "conda_environment": "",
     "conda_environment_boolean": false,
     "workspace": "",
-    "drillhole_group": {

```

Comment:
I think you might have deleted too much. Getting a crash with this missing
![image](https://github.com/MiraGeoscience/las-geoh5/assets/55204635/bb1b1c7f-8f7b-4748-9ae4-f39e28a8a673)

Reply:
yep, you're right.. whoops!